cloud-controller-manager/cmd/digitalocean-webhook-server/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Build the manager binary
+FROM golang:1.19 as builder
+
+WORKDIR /workspace
+
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+COPY ../../../../vendor/ vendor/
+
+
+# Copy the go source
+COPY  cloud-controller-manager/cmd/digitalocean-webhook-server/main.go main.go
+COPY  cloud-controller-manager/do cloud-controller-manager/do
+RUN go mod download
```

Comment:
You shouldn't need this. That's the whole point of copying the vendor dir.

Reply:
With the changes required to address this [comment](https://github.com/digitalocean/digitalocean-cloud-controller-manager/pull/579#discussion_r1100226476) in addition to removing that line, I run into this error:
``` => CACHED [builder 7/9] COPY  cloud-controller-manager/do cloud-controller-manager/do                                                                                                                                                                                         0.0s
 => ERROR [builder 8/9] RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -v -a -o doks_webhooks main.go                                                                                                                                                       0.3s
------
 > [builder 8/9] RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -v -a -o doks_webhooks main.go:
#14 0.259 go: inconsistent vendoring in /workspace:
#14 0.259       golang.org/x/sync@v0.1.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
#14 0.259       github.com/AanZee/goimportssort@v0.0.0-20200826100315-f4e155b80708: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
#14 0.259       github.com/dave/dst@v0.27.2: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
#14 0.259       golang.org/x/mod@v0.8.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
#14 0.259       golang.org/x/net@v0.5.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
#14 0.259       golang.org/x/sys@v0.5.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
#14 0.259       golang.org/x/term@v0.4.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
#14 0.259       golang.org/x/text@v0.6.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
#14 0.259       golang.org/x/tools@v0.5.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
#14 0.259       golang.org/x/net@v0.3.1-0.20221206200815-1e63c2f08a10: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
#14 0.259       golang.org/x/sync@v0.0.0-20220722155255-886fb9371eb4: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
#14 0.259       golang.org/x/sys@v0.3.0: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
#14 0.259       golang.org/x/term@v0.3.0: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
#14 0.259       golang.org/x/text@v0.5.0: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
#14 0.259       golang.org/x/tools@v0.2.0: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
#14 0.259 
#14 0.259       To ignore the vendor directory, use -mod=readonly or -mod=mod.
#14 0.259       To sync the vendor directory, run:
#14 0.259               go mod vendor
------
executor failed running [/bin/sh -c CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -v -a -o doks_webhooks main.go]: exit code: 1
```

I can only resolve it by adding  go mod tidy /  go mod vendor after copying the go source